client.js

```diff
@@ -200,52 +201,39 @@ class Client {
   /**
    * Request events based on the provided query parameters.
    *
-   * The `query.flags` field uses bitmask values similar to an enum:
+   * In CDP Studio, this corresponds to event log queries for the
+   * CDPLogger (or LogServer). The query parameters allow filtering by
+   * sender, data fields, code masks, and time ranges, among others.
    *
+   * The `query.flags` field uses bitmask values similar to an enum:
    *   0 = None  
    *   1 = NewestFirst  
    *   2 = TimeRangeBeginExclusive  
    *   4 = TimeRangeEndExclusive  
    *   8 = UseLogStampForTimeRange  
    * 
-   *      For additional information:
+   *   For additional information:
    *   https://cdpstudio.com/manual/cdp/cdp2sql/logmanager-eventquery.html#Flags-enum
    *   https://cdpstudio.com/manual/cdp/cdplogger/eventlogreader.html#cdp-event-code-flags
    *
-   * In addition, the user can simply supply the following properties in the query object:
-   *
-   *   - **senderConditions**: An array of sender strings (exact matches by default).
-   *   - **dataConditions**: An object where each key is a data field name and the value can be:
-   *       - A string (defaults to an exact match),
-   *       - An array of strings,
-   *       - An object (or array of objects) with properties:
-   *           - `value`: the string value to match,
-   *           - `matchType`: either `"exact"` (default) or `"wildcard"`.
-   *
-   * enum EventQuery::MatchType:
-   *   - Exact (0): The string must match exactly.
-   *   - Wildcard (1): The string may contain wildcards.
-   *
    * Example usage:
-   *
    *   // Filter for events with sender exactly "CDPLoggerDemoApp.InvalidLicense":
    *   { senderConditions: ["CDPLoggerDemoApp.InvalidLicense"] }
    *
    *   // Filter for events where the "Text" data field equals "Invalid or missing feature license detected.":
    *   { dataConditions: { "Text": "Invalid or missing feature license detected." } }
    *
-   *
    * @param {Object} query - A simple plain object representing the EventQuery.
```

Comment:
Where are the other fields documented (not just senderConditions and dataConditions). Seems most of query doc has gone missing?